### PR TITLE
Added CUAD metrics

### PR DESCRIPTION
**Pull request by bhavitvyamalik**
`EM`, `F1`, `AUPR`, `Precision@80%Recall`, and `Precision@90%Recall` metrics supported for CUAD

### CODE REVIEW

File: metrics/cuad/cuad.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+_DESCRIPTION = """
+This metric wrap the official scoring script for version 1 of the Contract
+Understanding Atticus Dataset (CUAD).
+Contract Understanding Atticus Dataset (CUAD) v1 is a corpus of more than 13,000 labels in 510
+commercial legal contracts that have been manually labeled to identify 41 categories of important
+clauses that lawyers look for when reviewing contracts in connection with corporate transactions.
+"""
+
+_KWARGS_DESCRIPTION = """
+Computes CUAD scores (EM, F1, AUPR, Precision@80%Recall, and Precision@90%Recall).
```
Comment (lhoestq) [inline metrics/cuad/cuad.py:40]: Maybe mention in the docstring that users can use multiple answers depending on a threshold on the confidence probability of each prediction ?

**Review by lhoestq — COMMENTED** [review]
Thanks !!
I added one last comment

File: metrics/cuad/cuad.py (hunk 1 above)
Comment (bhavitvyamalik) [reply metrics/cuad/cuad.py:40]: Added changes in docstring for `prediction_text`. Let me know if something is also required

**Review by lhoestq — APPROVED** [review]
perfect thanks :)

**Pull request merged**